src/abstractdataframe/show.jl

```diff
@@ -113,7 +91,9 @@ function compacttype(T::Type, maxwidth::Int=8, initial::Bool=true)
 
     maxwidth -= 1 # we will add "…" at the end
 
-    if T <: CategoricalValue
+    # This is only type display shortening so we
+    # are OK with any T whose name starts with Categorical here
```

Comment:
```suggestion
    # are OK with any T whose name starts with CategoricalValue here
```

CI fails due to `CategoricalValue` not being abbreviated to `Cat`. Not sure why.